apps/heft/src/plugins/Webpack/WebpackPlugin.ts

```diff
@@ -2,13 +2,19 @@
 // See LICENSE in the project root for license information.
 
 import * as webpack from 'webpack';
```

Comment:
In my test with the TSDoc branch, the dev server output looks like this:

```
C:\Git\tsdoc\playground>npm run start

> tsdoc-playground@0.1.0 start C:\Git\tsdoc\playground
> heft start

Project build folder is "C:\Git\tsdoc\playground"
Starting start
 ---- Compile started ---- 
 ---- Bundle started ---- 
[webpack] Using Webpack version 4.31.0
[typescript] Using TypeScript version 3.5.3
[typescript] Starting compilation in watch mode...
i ｢wds｣: Project is running at http://localhost:8080/
i ｢wds｣: webpack output is served from /
i ｢wds｣: Content not from webpack is served from C:\Git\tsdoc\playground
[copy-static-assets] Copied 6 static assets in 384ms
```
...but followed by 123 lines of additional verbose output that fills up the entire console window, causing the  `Project is running at http://localhost:8080/` line to scroll out of view.

Is there a way we could show this information at the very end of the initial log output?  The `http://localhost:8080/` URL is what you would normally want to click to open Chrome, and for newcomers it makes it clear that a server is running. It's pretty important information.

Reply:
Yeah I'd like to clean that up. We should plan on overhauling webpack logging, but that'll require some design and tinkering.